lib/ansible/plugins/lookup/first_found.py

```diff
@@ -15,17 +15,29 @@
         to the containing role/play/include/etc's location.
       - The list of files has precedence over the paths searched.
         i.e, A task in a  role has a 'file1' in the play's relative path, this will be used, 'file2' in role's relative path will not.
+      - Either a list of files (``_terms``) or a key `files` with a list of files is required for this plugin to operate.
```

Comment:
Assuming this is the same as modules, then I think you need `C(_terms)`, same comment applies else where